calculator.go

```diff
@@ -1,13 +1,96 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"fmt"
+	"math"
+	"strings"
+)
+
+// Add takes a list of numbers and returns the result of adding them together.
+func Add(a, b float64, nums ...float64) float64 {
+	var result float64 = a + b
```

Comment:
Great! And we could shorten this to just:

```suggestion
	result := a + b
```